phoenix-core/src/it/java/org/apache/phoenix/end2end/UpsertSelectIT.java

```diff
@@ -1242,11 +1384,13 @@ public void testRowTimestampColWithViewsIndexesAndSaltedTables() throws Exceptio
             tenantConn.commit();
         }
 
-        // Verify that the data upserted using the tenant view can now be queried using base table and the base table index
+        // Verify that the data upserted using the tenant view can now be queried using base table
+        // and the base table index
         Date upsertedDate;
         try (Connection conn = DriverManager.getConnection(getUrl(), props)) {
             // Query the base table
-            PreparedStatement stmt = conn.prepareStatement("SELECT * FROM  " + baseTable + " WHERE TENANT_ID = ? AND PK2 >= ? AND PK2 <= ? AND PK3 = ? ");
+            PreparedStatement stmt = conn.prepareStatement("SELECT * FROM  " + baseTable +
```

Comment:
why not use try-with-resources for statement also?

Reply:
Will do